.cirrus.yml

```diff
@@ -88,25 +97,35 @@ task:
     - pytest -v
 
 
-windows_task:
+task:
+  name: windows ${PY_VER}
   auto_cancellation: true
-  
+
   windows_container:
     image: cirrusci/windowsservercore:cmake
     os_version: 2019
-    
+
+  matrix:
+    # PR, only run on 3.8
+    - only_if: $CIRRUS_PR != ""
+      env:
+        PY_VER: "3.8"
+    # not a PR, run full suite
+    - only_if: $CIRRUS_PR == ""
+      env:
+        matrix:
+          PY_VER: "3.6"
+          PY_VER: "3.7"
+          PY_VER: "3.8"
   env:
-    matrix:
-      PY_VER: "3.6"
-      PY_VER: "3.7"
-    ANACONDA_LOCATION: $USERPROFILE\anaconda
-    PATH: $ANACONDA_LOCATION\Scripts;$ANACONDA_LOCATION;$PATH
-    # must set this to its default since it doesn't work in env variables
-    # see https://github.com/cirruslabs/cirrus-ci-docs/issues/423
-    CIRRUS_WORKING_DIR: C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build
-    # https://github.com/vispy/vispy/blob/v0.5.3/appveyor.yml#L44
-    VISPY_GL_LIB: $CIRRUS_WORKING_DIR\opengl32.dll
-    PYTHON_ARCH: 64
+      ANACONDA_LOCATION: $USERPROFILE\anaconda
```

Comment:
could you fix the indentation here?

Reply:
Before this, the file had really strange indentation, so do we want to go with 2 or 4 spaces?